### PR TITLE
[RFR] Menu change

### PR DESCRIPTION
**Pull request by psav**
Menu removal

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9d9e56647849ec71724b0f7be9b0f2b20b77706c:

`cfme/fixtures/pytest_selenium.py`:
- :red_circle: Line 22:1: **F401** *'ErrorInResponseException' imported but unused*
- :red_circle: Line 22:1: **F401** *'InvalidElementStateException' imported but unused*
- :red_circle: Line 22:1: **F401** *'InvalidSwitchToTargetException' imported but unused*
- :red_circle: Line 41:1: **F401** *'quit' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 06eda8beaf083338f30dc2b3c8e617397e487ac2:

`cfme/configure/tasks.py`:
- :red_circle: Line 11:1: **F401** *'paginator' imported but unused*
- :red_circle: Line 14:1: **F401** *'logger' imported but unused*
- :red_circle: Line 15:1: **F401** *'parsetime' imported but unused*

`cfme/fixtures/pytest_selenium.py`:
- :red_circle: Line 22:1: **F401** *'ErrorInResponseException' imported but unused*
- :red_circle: Line 22:1: **F401** *'InvalidElementStateException' imported but unused*
- :red_circle: Line 22:1: **F401** *'InvalidSwitchToTargetException' imported but unused*
- :red_circle: Line 41:1: **F401** *'quit' imported but unused*

`cfme/services/catalogs/orchestration_template.py`:
- :red_circle: Line 4:1: **F401** *'menu' imported but unused*

Hmmm
:frowning: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -132,74 +131,6 @@
...
-                        "cfg_settings_schedule_edit":
-                        lambda _: tb.select("Configuration", "Edit this Schedule")
-                    }
-                ]
-            }
-        ],
-    }
-)
-
-
```
Comment (mshriver) [inline cfme/configure/configuration/__init__.py]: :fire:

File: cfme/configure/red_hat_updates.py [hunk 2]
```diff
@@ -90,10 +89,11 @@ def make_update_button(text):
...
 
-appliances_table = lambda: version.pick({
-    version.LOWEST: CheckboxTable("//div[@id='form_div']/table[@class='style3']"),
-    '5.4': CheckboxTable("//div[@id='form_div']/table")
-})
+def appliances_table():
+    return version.pick({
+        version.LOWEST: CheckboxTable("//div[@id='form_div']/table[@class='style3']"),
+        '5.4': CheckboxTable("//div[@id='form_div']/table")
+    })
```
Comment (mshriver) [inline cfme/configure/red_hat_updates.py]: Old enough just just collapse into `return CheckboxTable("//div[@id='form_div']/table")`?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/configure/red_hat_updates.py [hunk 3]
```diff
@@ -91,10 +90,7 @@ def make_update_button(text):
 
 
 def appliances_table():
-    return version.pick({
-        version.LOWEST: CheckboxTable("//div[@id='form_div']/table[@class='style3']"),
-        '5.4': CheckboxTable("//div[@id='form_div']/table")
-    })
+    return CheckboxTable("//div[@id='form_div']/table[@class='style3']")
```
Comment (jkrocil) [inline cfme/configure/red_hat_updates.py]: I think in this case we want to keep the `5.4` one, not `version.LOWEST`.

**Pull request merged**